scripts/python/missing_followers.py

```diff
@@ -0,0 +1,41 @@
+import json
+import sys
+
+if len(sys.argv) == 3:
+ try:
+    print "Old:",sys.argv[1]
+    print "New:", sys.argv[2]
+    json_file_old = open(sys.argv[1])
+    connections_old = json.load(json_file_old)
+    json_file_new = open(sys.argv[2])
+    connections_new = json.load(json_file_new)
+    follower_old = []
+    follower_new = []
+    for user in connections_old['followers']:
+        follower_old.append(user)
+    for user in connections_new['followers']:
+        follower_new.append(user)
+
+    found_unfollower = False
```

Comment:
My mistake. This should be something like `found_still_following` and the other one should be `found_unfollower`. Last comment: just rebase and squash your commits down to 1 and then I'll approve! Thanks 😃 

So:
`found_unfollower` -> `found_still_following`
`found_overall` -> `found_unfollower`